kahuna/public/js/image/controller.js

```diff
@@ -41,6 +48,9 @@ image.controller('ImageCtrl', [
         };
 
         ctrl.image = image;
+
+        ctrl.imageDimensions = imageService(ctrl.image).dimensions;
```

Comment:
In Composer we've stayed away from "models" (rich objects that wrap data) and preferred a more functional approach using functions that transform plain data. I think it worked quite well, but happy to discuss.